src/api/app/views/webui2/webui/project/_tabs.html.haml

```diff
@@ -20,7 +20,7 @@
       // TODO: show it if the project have a staging_workflow attached when this feature will be released
       - if controller_name == 'staging_workflows'
         %li.nav-item
-          = tab_link('Staging', staging_workflow_path(project.staging))
+          = tab_link('Staging', request.url, controller_name == 'staging_workflows')
```

Comment:
[nitpick] Hm, and when we change the controller name? Is there no other way to do this?

Reply:
If you change the name of a controller you need to change this, right?

I don't see any other way to do it right now... because the tab needs to be shown in many actions of the StagingWorkflowController... and the only common thing is the controller

Reply:
> If you change the name of a controller you need to change this, right?

Yo but it feels unnatural to hardcode the controller name and need to change it when changing the actual controller name, no?

What about ``current_page?``
https://api.rubyonrails.org/v5.2.1/classes/ActionView/Helpers/UrlHelper.html#method-i-current_page-3F
?

Reply:
If we use **current_page?** then we have to use every single action page that needs to show the tab... Do you like it more?

Reply:
Hm, I see :thinking: 

When we do a helper method like
```
def current_controller(controller: nil, actions: [:index, :show, :edit])
  current_page?(controller: controller) && actions.any? { |action| current_page?(action:'index') }
end
```

Just brainstorming ...